web/src/emoji_picker.ts

```diff
@@ -649,7 +661,9 @@ function register_popover_events($popover: JQuery): void {
     $(".emoji-popover").on("keydown", process_keydown);
 }
 
-function get_default_emoji_popover_options(): Partial<tippy.Props> {
+function get_default_emoji_popover_options(
+    add_frequently_used_section: boolean,
```

Comment:
nit: `include_frequently_used_category` seems like a better name for this parameter? While some code refers to "sections", I think "Category" is more precise here. And I think `include` is a bit more precise of a verb, since "add" hints towards it being a mutation that is happening.